README.md

```diff
@@ -0,0 +1,31 @@
+# Agent Framework
+
+## .NET Core library for building Sovrin agents
+
+[![Build Status](https://dev.azure.com/streetcred/Agent%20Framework/_apis/build/status/Agent%20Framework%20-%20Build?branchName=master)](https://dev.azure.com/streetcred/Agent%20Framework/_build/latest?definitionId=10?branchName=master)
```

Comment:
Are we moving the CI at the same time?

Reply:
Up to you

Reply:
CI uses travis.ci which picked up right away. We will need to move CD and package deployment at some point. Package name spacing should also change from AgentFramework.* to Hyperledger.Aries.*
That means we have opportunity to reset the package version to 1.0. Let's have a separate discussion on that.